test-common/src/main/java/com/mentoring/database/DbUpdates.java

```diff
@@ -22,6 +25,15 @@ public static void setLastAccessValueByEmailAddress(Database connection, String
         waitForCacheUpdate();
     }
 
+/*    @Step("Updating production queue end time to be finished in the database.")
+    public static void setEndTimeByFactoryId(Database connection, String factoryId) {
+        connection.runUpdateLongColumnValue(
+                UPDATE_END_TIME_BY_FACTORY_ID,
+                connection.getSingleStringMapper(1, factoryId),
+                TableColumn.FACTORY__FACTORY_ID
```

Comment:
Params needed:
UPDATE_END_TIME_BY_FACTORY_ID
connection.getSingleStringMapper(2, factoryId)
connection.getSingleStringMapper(1, endTimeValue)

Please note that paramIndex in getSingleStringMapper is the index of the variable in the query (so factoryId is the 2nd and value is the 1st)